lib/super_sliver_list.dart

```diff
@@ -55,21 +55,17 @@ class _RenderSuperSliverList extends RenderSliverMultiBoxAdaptor {
 
   @override
   double? childScrollOffset(covariant RenderObject child) {
-    for (var c = firstChild; c != null; c = childAfter(c)) {
-      if (child == c) {
-        return super.childScrollOffset(child);
-      }
+    // If the child's layout offset is estimated, return the estimated scroll offset.
+    if (_layoutOffsetIsEstimated) {
+      final estimatedExtent = _estimateExtent();
+      return (indexOf(child) * estimatedExtent!)!;
     }
-    // Trying to query child offset of child that's not currently visible;
-    // Assume this is from viewPort.getOffsetToReveal, in which case we'll
-    // estimate the offset, but also remember the index and offset so that
-    // we can possibly correct scrollOffset in next performLayout call.
-    final offset = indexOf(child as RenderBox) * (_estimateExtent() ?? 0.0);
-    _estimatedOffsetChildIndex = indexOf(child);
```

Comment:
This change seems all wrong We remember _estimatedOffsetChildIndex so that later scroll position can be adjusted to show the child properly.